README.md

```diff
@@ -9,3 +9,22 @@ $ ./deploy/push-ecr-image.sh
 ```
 
 **NOTE:** The script attempts to auto-detect the target AWS region, but you may need to either set `AWS_REGION` in your environment or update your profile configuration with a `region` value.
+
+### Developer testing
+
+Currently, the application polling loop does not perform any actual work. Instead, the application can be tested by invoking a `run-studies.ts` script either locally or in the deployed container on ECS Fargate.
+
+#### Local testing steps
+
+Create a `.env` file from the example and populate with valid values. Note that the AWS values can be easily referenced in a deployed stack per [these environment variables](https://github.com/safeinsights/iac/blob/601155a55785996f736b0ed207945a9535c19371/secure-enclave/stack.ts#L273-L276) in the active task / task definition:
+
+```bash
+$ cp .env.example .env
+```
+
+Run the script (make sure you have AWS credentials for your target region created):
+
+```bash
+$ npm install
+$ npx tsx --env-file=.env ./src/lib/run-studies.ts
```

Comment:
it looks like we have `dotenv` in the dev dependencies, would you be interested in using `dotenv.config()` in the `run-studies.ts` file instead of having to load the env file like this?

Reply:
Thanks for the suggestion! I'm not familiar with `dotenv`. Will that then make the script assume a `.env` will be there in production? Right now we use `--env-file` for local testing, but the deployed container works via [these injected environment variables](https://github.com/safeinsights/iac/blob/98909628524e0535db6a22b9af7ba4363682794e/secure-enclave/stack.ts#L273-L276), so I just want to make sure if we use `dotenv` the latter will continue working :man_shrugging: .

Reply:
From a quick test, it seems like `dotenv` works if you:

* Have a `.env` file
* Don't have a `.env` file but the variables are set in the environment (e.g. it doesn't break the "happy path")

I'll push up a commit to use it and simplify the CLI invocation a bit. Thanks, @chrisbendel !